packages/cli/package.json

```
@@ -33,10 +33,8 @@
     "@types/node-fetch": "^2.5.5"
   },
   "scripts": {
-    "build": "yarn clean && yarn babel src --out-dir dist && cp -r ./src/commands/generate/templates/. ./dist/commands/generate/templates",
+    "build": "yarn cross-env NODE_ENV=production babel --delete-dir-on-start src --out-dir dist --copy-files --no-copy-ignored",
```

Not a train smash, but I have trouble mentally parsing `--delete-dir-on-start src` since it looks like that is the directory to delete, but it's actually the files to compile (I know they're the same.)

```suggestion
    "build": "babel --out-dir dist --env-name PRODUCTION --delete-dir-on-start --copy-files --no-copy-ignored src",
```

I also found the `--env-name` flag in babel-cli!

---

```bash
  --env-name [name]                           The name of the 'env' to use when loading configs and plugins. Defaults to the value of
                                              BABEL_ENV, or else NODE_ENV, or else 'development'.

```

---

Agreed these are painful to read. Feel the same and probably spent too much on that myself before getting reactions. Anyway, for both of these you’re suggesting I try:
1. re-ordering flags to make `--delete-dir-on-start` more understandable. I could try: 
`yarn cross-env NODE_ENV=production babel src --out-dir dist --delete-dir-on-start --copy-files --no-copy-ignored`
2. See if `--env-name` flag can replace `cross-env` package implementation

Correct?

---

Yup! I actually prefer your suggestion to add it to the front!

---

Babel Docs are just, grrrr.

`--env-name` does not allow us to set the variable, apparently. It’s just for choosing an env preset, which doesn’t really work with our current setup. But in the future this could be handy — could have settings specific to BABLE_ENV separate from those for NODE_ENV. Anyway, I actually found in some earlier version of the Babel docs where they just say to "use `cross-env`”. There you have it.